core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/groupBy.kt

```diff
@@ -21,14 +36,289 @@ import kotlin.reflect.KProperty
+     *
+     * ### Aggregate [GroupBy] into [DataFrame]
+     *
+     * {@include [Indent]}
+     * [GroupBy][GroupBy]`.`[**`concat`**][GroupBy.concat]**`() `**
+     *
+     * {@include [Indent]}
+     * `| `__`.`__[**`concatWithKeys`**][GroupBy.concatWithKeys]**`() `**
+     *
+     * {@include [Indent]}
+     * `| `__`.`__[**`into`**][GroupBy.into]**`(`**`column: `[`String`][String]**`) `**`  \[  `**`{  `**`rowExpression: `[`RowExpression`][RowExpression]**` } `**`]`
+     *
+     * {@include [Indent]}
+     * `| `__`.`__[**`values`**][Grouped.values]**`  {  `**`valueColumns: `[`ColumnsSelector`][ColumnsSelector]**` }`**
+     *
+     * {@include [Indent]}
+     * `| `__`.`__[**`count`**][Grouped.count]**`() `**
+     *
+     * {@include [Indent]}
+     * `| `__`.`__[**`aggregate`**][Grouped.aggregate]**`  {  `**`aggregations: `[`AggregateDsl`][AggregateDsl]**` }`**
+     *
+     * {@include [Indent]}
+     * `| `__`.`__[<aggregation_statistic>][AggregationStatistics]
+     *
+     * ### Pivot [GroupBy] into [PivotGroupBy] and reduce / aggregate it
+     *
+     * {@include [Indent]}
+     * `| `__`.`__[**`pivot`**][GroupBy.pivot]**`  {  `**`columns: `[`ColumnsSelector`][ColumnsSelector]**` }`**
+     *
+     * {@include [Indent]}
+     * `    \[ `__`.`__[**`default`**][PivotGroupBy.default]**`(`**`defaultValue`**`) `**`]`
+     *
+     * {@include [Indent]}
+     * `| `__`.`__[<pivot_reducer>][PivotGroupByDocs.Reducing]
+     *
+     * {@include [Indent]}
+     * `| `__`.`__[<pivot_aggregator>][PivotGroupByDocs.Aggregation]
+     *
+     * Check out [PivotGroupBy Grammar][PivotGroupByDocs.Grammar] for more information.
+     */
+    interface Grammar
+
+    /**
+     * {@comment Version of [SelectingColumns] with correctly filled in examples}
+     * @include [SelectingColumns] {@include [SetGroupByOperationArg]}
+     */
+    interface GroupBySelectingOptions
+
+    /**
+     * ### [GroupBy] aggregation statistics
+     * * [count][Grouped.count]
+     * * [max][Grouped.max]/[maxOf][Grouped.maxOf]/[maxFor][Grouped.maxFor]
+     * * [min][Grouped.min]/[minOf][Grouped.minOf]/[minFor][Grouped.minFor]
+     * * [sum][Grouped.sum]/[sumOf][Grouped.sumOf]/[sumFor][Grouped.sumFor]
+     * * [mean][Grouped.mean]/[meanOf][Grouped.meanOf]/[meanFor][Grouped.meanFor]
+     * * [std][Grouped.std]/[stdOf][Grouped.stdOf]/[stdFor][Grouped.stdFor]
+     * * [median][Grouped.median]/[medianOf][Grouped.medianOf]/[medianFor][Grouped.medianFor]
+     * * [percentile][Grouped.percentile]/[percentileOf][Grouped.percentileOf]/[percentileFor][Grouped.percentileFor]
+     */
+    interface AggregationStatistics
+
+    /**
+     * ### [GroupBy] transformations
+     *
+     * A [GroupBy] can be transformed into a new [GroupBy] using one of the following methods:
+     * * [sortByGroup][GroupBy.sortByGroup] / [sortByGroupDesc][GroupBy.sortByGroupDesc] — sorts the **order
```

Comment:
since these first 4 have a different default for asc/desc, could you mention something about which order is used by default? Or do you think this is clear enough when shown like this?